src/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Compression App</h1>
+    <br/>
+    <form action="http://localhost:5700/compress">
+      <input type="file" />
+      <br>
+      <label for="#compressionType">Select compression type</label>
```

Comment:
The `for` attribute in the label should not have a `#` symbol. It should match the `id` of the associated input element. Change `for="#compressionType"` to `for="compressionType"`.